src/iuse_actor.cpp

```diff
@@ -4255,10 +4255,9 @@ int weigh_self_actor::use( player &p, item &, bool, const tripoint & ) const
     // this is a weight, either in kgs or in lbs.
     double weight = convert_weight( p.get_weight() );
     if( weight > convert_weight( max_weight ) ) {
-        popup( "ERROR TOO HEAVY" );
+        popup( _( "ERROR: Max weight of %.0f %s exceeded" ), weight, weight_units() );
```

Comment:
Shouldn't this be **max_weight**? - the idea being that they wouldn't know their actual weight, but they would know its over what ever the maximum the scale can handle.